### PR TITLE
Added middle section 

### PR DESCRIPTION
**Pull request by spydaspider**
# Middle Section Of the Mobile Version of my portfolio.
  # The middle section branch contains code for the rest of the portfolio pages which can be found in the images below.
  
- [ ] Bug fix (unexpected generic font-family).
- [ ] New feature ( font-family; Poppins, Arial, Helvetica, sans-serif;)
  - [ ] Run local stylelint config command(npx stylelint "**/*.{css,scss}" --fix  after changing the font.
# Checklist
 [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,6 +47,164 @@ <h2>LET'S CONNECT</h2>
         <li><a href=""><img src="images/vectorhand.png" alt = "image"/></a></li>
         </ul>
         </div>
+        <section class = "second-section-wrapper">
```
Comment (ShahierNashaat) [inline index.html:50]: - [ ] Kindly be noticed that the container of the works section is a `section` tag.
- [ ] And the main container of the projects has a `display:grid`.

File: index.html [hunk 2]
```diff
@@ -47,6 +47,164 @@ <h2>LET'S CONNECT</h2>
...
+            <li><a href = "" class = "f5-base-link-text">css</a></li>
+            <li><a href = "" class = "f5-base-link-text">javascript</a></li>
+          </ul>
+          </div>
+          <div class = "base-button">
+            <a href = "">See Project</a>
+          </div>
+          </section>
+        </section>
+        <div class = "grid-section-wrapper">
```
Comment (ShahierNashaat) [inline index.html:155]: - [ ] Kindly be noticed that the container of the about me section is a `section` tag.

File: index.html [hunk 3]
```diff
@@ -47,6 +47,164 @@ <h2>LET'S CONNECT</h2>
...
+            </div>
+            <div class = "item2">
+              <a href = ""><img src = "images/Frame9.png" alt = "image"></a>
+             </div>
+             <div class = "item3">
+              <a href = ""><img src = "images/Frame10.png" alt = "image"></a>
+             </div>
+             <div class = "item4">
+              <a href = ""><img src = "images/Frame11.png" alt = "image"></a>
+             </div>
```
Comment (ShahierNashaat) [inline index.html:192]: - [ ] Kindly be noticed that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @spydaspider , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please fix your design as there is a space in the right of the design.
Screenshot from 2022-05-28 15-25-09

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (spydaspider) [reply index.html:192]: Thank you Sir! I will 
attend to the errors at once.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️
 - [ ] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
 - [ ] Kindly remove one Author in the readme since you are alone on this project
 

Capturer
  
image
 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by spydaspider** [comment]
Thanks for the feedback!

**Comment by spydaspider** [comment]
Awesome, thanks!

**Pull request merged**